site/content/extend/plugins/example-plugins.md

```diff
@@ -1,9 +1,29 @@
 ---
-draft: true
 title: Example Plugins
 date: 2017-10-26T17:54:54-05:00
 subsection: Plugins
 weight: 40
 ---
 
-Construction
+# Example Plugins
+
+## Server "Hello, world!"
+
+To get started extending server-side functionality with plugins, take a look at our [server "Hello, world!" tutorial](../server/hello-world).
+
+## Web App "Hello, world!"
+
+To get started extending browser-side functionality with plugins, take a look at our [web app "Hello, world!" tutorial](../webapp/hello-world).
+
+## JIRA
+
+The [JIRA plugin for Mattermost](https://github.com/mattermost/mattermost-plugin-jira) creates a webhook that your JIRA server can use to post messages to Mattermost when issues are created:
```

Comment:
@jwilander might also be a good time to actually review this repo (https://github.com/mattermost/mattermost-plugin-jira). The code is all the same as it was before, but I added a bunch of tests, the Makefile, Travis-CI, and Codecov.

Reply:
Already started looking at it as an example for how to structure the zoom plugin's backend 👍